### DISCUSSION TITLE
Is it possible to make Just (video)Player recognized as an External Player in Jellyfin for FireTV?

### DISCUSSION THREAD

**Discussion by averagejooe** [jellyfin/jellyfin-androidtv#3047] [General]
Sorry if this question is not relevant to Jellyfin. I don't know why Justplayer and Kodi for example is not recognized by my Firestick 4K running Jellyfin for Fire as an external player. Some other player like Mova and unofficial Kodi instance did get recognized. I guess it is kinda strange because Android TV counterparts doesn't have this issue.

I choose Justplayer for the ability to stream DV without the embed subtitle not loaded bugs, which still present in JF Exoplayer.
Again, I apologize if the question is not relevant because I can't pinpoint where the fault lies; it could be Just (Video) Player, JF, or limitation from Firestick 4K itself.

**Comment by nielsvanvelzen** [comment]
In short;
When using an external player the app just tells the system "I want to play this video" and the system needs to figure out how to grant that wish. If certain apps don't show up they did not tell the system that they are able to open video files.
